src/lib/cli/rule-generator.ts

```diff
@@ -60,7 +60,7 @@ export type NewRule = {
     /** Category of the new rule */
     category: string,
     /** Description of the new rule */
-    description: string,
+    description: object,
```

Comment:
@molant  It returns an object that has a property of `string`. I edited the commit and added a `SafeString` type to be more specific. But unless I make this `string` property optional in the type, an error of ` Type 'hbs.SafeString' is not assignable to type 'SafeString'` always shows. Any suggestion on better solving that?